nifi-nar-bundles/nifi-spark-bundle/nifi-livy-processors/src/test/java/org/apache/nifi/processors/livy/TestExecuteSparkInteractiveSSL.java

```diff
@@ -109,13 +109,15 @@ private static TestServer createServer() throws IOException {
     public void testSslSparkSession() throws Exception {
         addHandler(new LivyAPIHandler());
 
-        runner.enqueue("print \"hello world\"");
+        String code = "print \"hello world\" //'?!<>[]{}()$&*=%;.|_-\\";
```

Comment:
I think that adding a UT for the non-SSL case is enough, isn't it? There is no difference among SSL and non-SSL about escaping and the content IIUC.

Reply:
Removed.